com.github.tenderowl.frog.json

```diff
@@ -0,0 +1,96 @@
+{
+    "app-id" : "com.github.tenderowl.frog",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "41",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "frog",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        "leptonica.json",
+        "tesseract.json",
+        "libportal.json",
+        "zbar.json",
```

Comment:
the content of these is missing now.

Reply:
That's why I put them inside this repo. So, does it mean I should avoid putting any other files here, but put all the modules inside manifest instead?

Reply:
that's exactly what I meant